Contributing.md

```diff
@@ -19,13 +19,14 @@ Use this handy checklist to make sure your new instructions are fully supported:
  - [ ] Effects handled in src/ir/effects.h
  - [ ] Precomputing handled in src/passes/Precompute.cpp
  - [ ] Hashing and comparing in src/ir/ExpressionAnalyzer.cpp
- - [ ] Parsing added in scripts/gen-s-parser.py and src/wasm/wasm-s-parser.cpp
+ - [ ] Parsing added in scripts/gen-s-parser.py, src/wasm-s-parser.h and src/wasm/wasm-s-parser.cpp
  - [ ] Printing added in src/passes/Print.cpp
  - [ ] Decoding added in src/wasm-binary.h and src/wasm/wasm-binary.cpp
  - [ ] Binary writing added in src/wasm-stack.h and src/wasm/wasm-stack.cpp
+ - [ ] Support added in various classes inheriting OverridenVisitor (and possibly other non-OverridenVisitor classes as necessary)
```

Comment:
Is there something we can recommend grepping for to audit the non-OverriddenVisitor classes?

`Overriden` => `Overridden`

Reply:
Not sure. Basically you should check each pass to be sure. For most non-control-flow instructions there shouldn't be many though.

Reply:
Ok, sgtm :+1: